homework4/4.1/4.1/Substraction.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _4._1;
+
+public class Substraction : Operation
```

Comment:
```suggestion
public class Subtraction : Operation
```
Я тоже всю жизнь неправильно писал